### DISCUSSION TITLE
自定义java写的插件怎么编排啊。基于那个条件判断, 编排了两个插件，只执行了一个就往下了。

### DISCUSSION THREAD

**Discussion by objectxz** [apache/apisix#7664] [Q&A]
于那个条件判断, 编排了两个插件，只执行了一个就往下了。
有没有相应的自定义Java插件的编排文档。

**Comment by tzssangglass** [comment]
the filter plugin feature can not work on java plugin runner

**Reply by 18119634869** [reply]
您描述的问题解决了嘛？我也想知道0.0
